docs/operations/metrics.md

```diff
@@ -264,40 +264,42 @@ If the JVM does not support CPU time measurement for the current thread, `ingest
+| `taskSlot/idle/count`                    | Number of idle task slots per emission period. This metric is only available if the `TaskSlotCountStatsMonitor` module is included.                                                                                | `category`|Varies|
+| `taskSlot/used/count`                    | Number of busy task slots per emission period. This metric is only available if the `TaskSlotCountStatsMonitor` module is included.                                                                                | `category`|Varies|
+| `taskSlot/lazy/count`                    | Number of total task slots in lazy marked Middle Managers and Indexers per emission period. This metric is only available if the `TaskSlotCountStatsMonitor` module is included.                                   | `category`|Varies|
+| `taskSlot/blacklisted/count`             | Number of total task slots in blacklisted Middle Managers and Indexers per emission period. This metric is only available if the `TaskSlotCountStatsMonitor` module is included.                                   | `category`|Varies|
+| `worker/task/failed/count`               | Number of failed tasks run on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included, and is only supported for Middle Manager nodes.     | `category`, `workerVersion`|Varies|
+| `worker/task/success/count`              | Number of successful tasks run on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included, and is only supported for Middle Manager nodes. | `category`,`workerVersion`|Varies|
+| `worker/taskSlot/idle/count`             | Number of idle task slots on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included, and is only supported for Middle Manager nodes.      | `category`, `workerVersion`|Varies|
+| `worker/taskSlot/total/count`            | Number of total task slots on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.                                                     | `category`, `workerVersion`|Varies|
+| `worker/taskSlot/used/count`             | Number of busy task slots on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.                                                      | `category`, `workerVersion`|Varies|
+| `worker/task/assigned/count`             | Number of tasks assigned to an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.                                                                 |`dataSource`|Varies|
```

Comment:
Please revert these formatting changes. If IDEA seems to be doing this automatically, you can turn off the following feature:

<img width="730" alt="Screenshot 2024-06-18 at 9 42 55 AM" src="https://github.com/apache/druid/assets/18635897/cd68de89-67f8-4f3c-a8f3-9905a3035195">